### DISCUSSION TITLE
Add option to use content of open editor tabs

### DISCUSSION THREAD

**Discussion by chriamue** [Christopher-Hayes/vscode-chatgpt-reborn#29] [Ideas]
It would be great to have an option that allows us to use the content of all open editor tabs, similar to the existing "Use Editor Selection" feature.

This would provide users with the flexibility to include relevant information from multiple tabs when making API calls, resulting in better context-aware suggestions and responses from ChatGPT. In addition, it could save time and effort, as users would no longer need to manually copy and paste the content from various tabs to include in the API call.

Here is a draft commit:
https://github.com/Christopher-Hayes/vscode-chatgpt-reborn/compare/develop...chriamue:vscode-chatgpt-reborn:feature/files-as-context

**Comment by Christopher-Hayes** [comment]
Sorry I didn't realize I wasn't getting notifications for Discussions. That's a cool idea @chriamue, I'll be sure to try that out and see what makes the most sense for the UI.
